src/Components/FilmCard.css

```diff
@@ -0,0 +1,117 @@
+.film-card{
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  position: relative;
+  cursor: pointer;
+}
+
+.film-card:hover::after{
+  z-index: 2;
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0));
+}
+
+.title{
+  display: none;
+  margin:0;
+}
+
+.film-card p {
+  display: none;
+}
+
+.film-img{
+  height: 100%;
+  width: 100%;
+}
+
+.film-card:hover .title{
+  display: block;
+  color: white;
+}
+
+.film-card:hover .overview{
+  display: -webkit-box;
+  -webkit-line-clamp: 2; /* Number of lines to show */
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  text-overflow: ellipsis; 
+  color: white;
+}
+
+.film-card-text-wrap{
+  display: flex;
+  flex-direction: column;
+  z-index: 3;
+  position: absolute;
+  padding: 10px;
+}
+
+.overview-container p{
+  display: none;
+}
+
+.link-film-card{
+  display: none;
+}
+
+.film-card:hover .link-film-card{
+  display: flex;
+  color: white;
+}
+
+/* in this link I found some help from Fd-Web-desingn on how to do the arrows animation
+https://www.youtube.com/watch?app=desktop&v=hubxcpovlC4 */
+.arrows{
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding-bottom: 5px;
+}
+
+.link-film-card {
+  position: relative;
+  width:  20px;
+  height: 20px;
+  border-bottom: 4px solid #fff;
+  border-right: 4px solid #fff;
+  transform: rotate(-45deg);
+  animation: arrow-load 3s ease infinite;
+}
+
+.link-film-card:nth-child(1){
+  left: 30px;
+  animation-delay: -0.4s;
+}
+
+.link-film-card:nth-child(2){
+  animation-delay: -0.2s;
+}
+
+.link-film-card:nth-child(3){
+  right: 30px;
+}
+
+@keyframes arrow-load
+{
+  0%
+  {
+    opacity: 0;
+    transform: rotate(-45deg) translate(-60px, -60px)
+  }
+  50%
+  {
+    opacity: 1;
+  }
+  100%
+  {
+    opacity: 0;
+    transform: rotate(-45deg) translate(60px, 60px)
+  }
+}
```

Comment:
I think it looks cool, but I think it's really confusing with a moving button 😅 Consider making the whole card into a link if you want to keep this animation

Reply:
I changed the button style and the link is working now.